.travis.yml

```diff
@@ -17,3 +19,8 @@ before_install:
 script:
 - make check_license style unused test lint check_assets
 - git diff --exit-code
+- ./fuzzit.sh
```

Comment:
The first comment in `fuzzit.sh` says it should be called with either "fuzzing" or "local-regression".

Reply:
Ah. I see from the bottom of `fuzzit.sh` that it will auto-select "local-regression" for pull-requests and "fuzzing" for everything else. Perhaps this should be encoded in a special argument? `./fuzzit.sh auto-select-for-travis`, so there's no doubt?

Reply:
Removed. This is old docs.

Reply:
Yes probably we will add something like this in the near future but for now I'm trying to avoid CI specific logic inside the CLI.